source/sgp_mode/GenomeLibrary.h

```diff
@@ -256,6 +256,16 @@ class ProgramBuilder : emp::vector<sgpl::Instruction<Spec>> {
     Add("Nand", 0, 0, 0);
     Add("SharedIO");
   }
+
+  void AddSteal(){
+    
+    Add("Steal");
+    //std::cout << "Adding Steal" << std::endl;
```

Comment:
remove comment
Also why are these needed? Why can't you just call "Add("Steal") in the create programs?